yo polymer:seed error. "AssertionError: Trying to copy from a source that does not exist"

I'm getting a stacktrace when I attempt to use the polymer:seed generator.

```
events.js:141
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: /Users/212422863/apps/node-v4.2.1-darwin-x64/lib/node_modules/generator-polymer/seed/templates/seed-element/.gitignore
    at EditionInterface.exports._copySingle (/Users/212422863/apps/node-v4.2.1-darwin-x64/lib/node_modules/generator-polymer/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/Users/212422863/apps/node-v4.2.1-darwin-x64/lib/node_modules/generator-polymer/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy.js:23:17)
    at module.exports.yeoman.generators.Base.extend.seed (/Users/212422863/apps/node-v4.2.1-darwin-x64/lib/node_modules/generator-polymer/seed/index.js:125:15)
    at /Users/212422863/apps/node-v4.2.1-darwin-x64/lib/node_modules/generator-polymer/node_modules/yeoman-generator/lib/base.js:429:16
    at processImmediate [as _immediateCallback] (timers.js:368:17)
```
Versions:

OS: Mac Yosemite
node: v4.2.1
npm: 2.14.7
yo: 1.4.8

Listing of the directory generator-polymer/seed/templates/seed-element/

```
drwxr-xr-x  10 212422863 Domain Users   340B Oct 20 16:13 .
drwxr-xr-x   3 212422863  Domain Users   102B Oct 16 00:10 ..
-rw-r--r--   1 212422863  Domain Users    17B Oct 16 00:10 .npmignore
-rw-r--r--   1 212422863  Domain Users   1.6K Oct 16 00:10 README.md
-rw-r--r--   1 212422863  Domain Users   646B Oct 16 00:10 bower.json
drwxr-xr-x   3 212422863 Domain Users   102B Oct 16 00:10 demo
-rwxr-xr-x   1 212422863  Domain Users   1.1K Oct 16 00:10 hero.svg
-rw-r--r--   1 212422863  Domain Users   1.1K Oct 16 00:10 index.html
-rw-r--r--   1 212422863  Domain Users   4.0K Oct 16 00:10 seed-element.html
drwxr-xr-x   4 212422863Domain Users   136B Oct 20 16:13 test
```

FWIW I freshly installed npm on my Windows 10 home machine and got the same output.

```
C:\Users\Ross\foo>yo polymer:seed my-ele
? ==========================================================================
? ==========================================================================
We're constantly looking for ways to make yo better!
May we anonymously report usage statistics to improve the tool over time?
More info: https://github.com/yeoman/insight & http://yeoman.io
========================================================================== Yes

     _-----_
    |       |    .--------------------------.
    |--(o)--|    | Out of the box I include |
   `---------´   |  Polymer's seed-element. |
    ( _´U`_ )    '--------------------------'
    /___A___\
     |  ~  |
   __'.___.'__
 ´   `  |° ´ Y `

? What is your GitHub username? 212422863
? Would you like to include web-component-tester? Yes

events.js:141
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: C:\Users\Ross\AppData\Roaming\npm\node_modules\generator-polymer\seed\templates\seed-element\.gitignore
    at EditionInterface.exports._copySingle (C:\Users\Ross\AppData\Roaming\npm\node_modules\generator-polymer\node_modules\yeoman-generator\node_modules\mem-fs-editor\actions\copy.js:45:3)
    at EditionInterface.exports.copy (C:\Users\Ross\AppData\Roaming\npm\node_modules\generator-polymer\node_modules\yeoman-generator\node_modules\mem-fs-editor\actions\copy.js:23:17)
    at module.exports.yeoman.generators.Base.extend.seed (C:\Users\Ross\AppData\Roaming\npm\node_modules\generator-polymer\seed\index.js:125:15)
    at C:\Users\Ross\AppData\Roaming\npm\node_modules\generator-polymer\node_modules\yeoman-generator\lib\base.js:429:16
    at processImmediate [as _immediateCallback] (timers.js:368:17)

```

I checked my npm version (2.14.2) and node version (v4.0.0)

```
events.js:141
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: C:\Users\wilbert\AppData\Roaming\npm\node_modules\generator-polymer\seed\templates\seed-element\.gitignore
    at EditionInterface.exports._copySingle (C:\Users\wilbert\AppData\Roaming\npm\node_modules\generator-polymer\node_modules\yeoman-generator\node_modules\mem-fs-editor\actions\copy.js:45:3)
    at EditionInterface.exports.copy (C:\Users\wilbert\AppData\Roaming\npm\node_modules\generator-polymer\node_modules\yeoman-generator\node_modules\mem-fs-editor\actions\copy.js:23:17)
    at module.exports.yeoman.generators.Base.extend.seed (C:\Users\wilbert\AppData\Roaming\npm\node_modules\generator-polymer\seed\index.js:125:15)
    at C:\Users\wilbert\AppData\Roaming\npm\node_modules\generator-polymer\node_modules\yeoman-generator\lib\base.js:429:16
    at processImmediate [as _immediateCallback] (timers.js:371:17)

wilbert@WILBERTPC /d/Projects/polymer/aj-autocomplete
$ npm --version
2.14.2

wilbert@WILBERTPC /d/Projects/polymer/aj-autocomplete
$ node --version
v4.0.0
```

--- a/seed/index.js
+++ b/seed/index.js
@@ -116,7 +116,7 @@ module.exports = yeoman.generators.Base.extend({
 
     // Handle bug where npm has renamed .gitignore to .npmignore
     // https://github.com/npm/npm/issues/3763
-    if (this.fs.exists('.npmignore')) {
+    if (this.fs.exists(this.templatePath('.npmignore'))) {
       this.fs.copy(
         this.templatePath('.npmignore'),
         this.destinationPath('.gitignore')

